requirements/default.txt

```diff
@@ -1,3 +1,4 @@
 celery>=5.2.3,<6.0
-django-timezone-field>=4.2.3
+django-timezone-field>=5.0
+tzdata
 python-crontab>=2.3.4
```

Comment:
backports.zoneinfo;python_version<"3.9" should this be added as well?

Reply:
This is added by django-timezone-field already, though I agree it wouldn't hurt to make explicit.

(especially as we do import it ourselves)